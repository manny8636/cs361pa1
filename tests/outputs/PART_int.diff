--- outputs/PART_int.txt	2022-02-14 17:19:49.456418244 -0500
+++ expected/PART_int.txt	2022-02-13 17:48:24.370294554 -0500
@@ -0,0 +1,2 @@
+INTEGER: 123
+Success!
