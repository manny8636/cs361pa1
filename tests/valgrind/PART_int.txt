<<<<<<< HEAD
==146635== Memcheck, a memory error detector
==146635== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==146635== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==146635== Command: ../json -i ../data/int.txt
==146635== 
Parsing ../data/int.txt failed
==146635== 
==146635== HEAP SUMMARY:
==146635==     in use at exit: 4 bytes in 1 blocks
==146635==   total heap usage: 4 allocs, 3 frees, 16,860 bytes allocated
==146635== 
==146635== LEAK SUMMARY:
==146635==    definitely lost: 4 bytes in 1 blocks
==146635==    indirectly lost: 0 bytes in 0 blocks
==146635==      possibly lost: 0 bytes in 0 blocks
==146635==    still reachable: 0 bytes in 0 blocks
==146635==         suppressed: 0 bytes in 0 blocks
==146635== Rerun with --leak-check=full to see details of leaked memory
==146635== 
==146635== For lists of detected and suppressed errors, rerun with: -s
==146635== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
=======
==149594== Memcheck, a memory error detector
==149594== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==149594== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==149594== Command: ../json -i ../data/int.txt
==149594== 
==149594== 
==149594== HEAP SUMMARY:
==149594==     in use at exit: 88 bytes in 1 blocks
==149594==   total heap usage: 4 allocs, 3 frees, 8,872 bytes allocated
==149594== 
==149594== LEAK SUMMARY:
==149594==    definitely lost: 88 bytes in 1 blocks
==149594==    indirectly lost: 0 bytes in 0 blocks
==149594==      possibly lost: 0 bytes in 0 blocks
==149594==    still reachable: 0 bytes in 0 blocks
==149594==         suppressed: 0 bytes in 0 blocks
==149594== Rerun with --leak-check=full to see details of leaked memory
==149594== 
==149594== For lists of detected and suppressed errors, rerun with: -s
==149594== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
>>>>>>> 149606adc8b3f0cb79ed9f7c590375c37f62be46
