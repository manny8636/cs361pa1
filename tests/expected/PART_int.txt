INTEGER: 123
Success!
